util/gather-docs.ts

```diff
@@ -0,0 +1,103 @@
+import { readdir, stat } from "fs/promises";
+import * as path from "path";
+import { spawn } from "child_process";
+
+function convertWindowsToLinuxPath(windowsPath) {
+    // Normalize the path to handle mixed slashes
+    const normalizedPath = path.normalize(windowsPath);
+    // Convert to POSIX format
+    return path.posix.join(...normalizedPath.split(path.sep)).split("G:/").join("/mnt/g/");
```

Comment:
I couldn't find where this path for a G drive was defined?

Reply:
Yea very specific to my machine (for now).

Reply:
FWIW - I keep my "dev" folders in a ReFS virtual disk mounted as "G" on all my machines:
![image](https://github.com/user-attachments/assets/e5720c9b-f6f8-46f0-b393-1f10e3a78ad7)